custom/icds_reports/ucr/tests/test_migration_form.py

```diff
@@ -14,8 +14,10 @@ def test_migration_form(self):
         self._test_data_source_results(
             'migration_form_v31895',
             [{
+                "date_left": None,
```

Comment:
could you add a test form to check whether this new field added `date_left` is actually fetching the data from form.